### DISCUSSION TITLE
Suggestion: Add ability to add all livewire components as block element

### DISCUSSION THREAD

**Discussion by vlados** [Z3d0X/filament-fabricator#72] [Ideas]
What if we can add any livewire component as block element? Maybe also we can add properties to this elements?

**Comment by Z3d0X** [comment]
What do you mean? what are you trying to do?

**Reply by vlados** [reply]
If I have a livewire component <livewire:counter /> in my app to be able from fabricator to add it to any page

**Reply by Z3d0X** [reply]
What's preventing you from doing that? What have you tried?

**Reply by vlados** [reply]
Because from the documentation I need to create a PageBlock and inside to put the livewire component
I am suggesting all livewire components to be able to be added as a block

**Reply by Z3d0X** [reply]
The "PageBlock" is a blade component, you can place a livewire components inside this blade view
